scripts/lib/debug.ts

```diff
@@ -0,0 +1,69 @@
+import { fetch } from 'cross-fetch';
+import deepmerge from 'deepmerge';
+import { adapterForRequest, decodeRequest, processRequest, type Adapter, type Request } from '../../src';
+
+export const debugAdapter = async (adapter: Adapter, options?: { accumulateToPath?: boolean; rowLimit?: number }) => {
+    // Fetch requests from Datasette.
+    const adapterClauses = adapter.endpointUrls.map((u) =>
+        u instanceof RegExp
+            ? `endpointUrl regexp '${
+                  // JS escapes slashes in regexes, but sqlite-regex doesn't accept that.
+                  u.source.replace(/\\\//g, '/')
+              }'`
+            : `endpointUrl = '${u}'`
+    );
+    const whereClause = `endpointUrl is not null and (${adapterClauses.join(' or ')})`;
+
+    const requests: Request[] = [];
+    let nextUrl = `https://data.tweasel.org/data/requests.json?_shape=objects&_where=${encodeURIComponent(
+        whereClause
+    )}&_json=headers&_json=cookies&_nocol=initiator&_nocol=platform&_nocol=runType&_nofacet=1&_nosuggest=1&_nocount=1&_size=${
+        options?.rowLimit || 'max'
+    }`;
+    while (nextUrl && options?.rowLimit && requests.length < options?.rowLimit) {
```

Comment:
Otherwise, you can never request `max` as the row limit.

```suggestion
    while (nextUrl && (!options?.rowLimit || requests.length < options.rowLimit)) {
```